rest-api/dao/biobank_order_dao.py

```diff
@@ -201,6 +220,37 @@ def _update_participant_summary(self, session, obj):
       setattr(participant_summary, status_field, status)
       setattr(participant_summary, status_field + 'Time', time)
 
+  def _get_non_cancelled_biobank_orders(self, session, participantId):
+    # look up latest order without cancelled status
+    return session.query(BiobankOrder).filter(BiobankOrder.participantId ==
+                                              participantId).filter(or_(BiobankOrder.orderStatus !=
+                                                                    BiobankOrderStatus.CANCELLED,
+                                                                    BiobankOrder.orderStatus == None
+                                                                    )).order_by(
+      BiobankOrder.created.desc()).all()
```

Comment:
I would get rid of the desc(); that way below you're going through orders in ascending timestamp order, which is the right behavior.